### ISSUE TITLE
Need option to choose between %20 and + for spaces

### ISSUE THREAD

**Issue by indirect** [rs/SafariOmnibar#15]
I discovered one more issue with keyword searches: some sites expect spaces encoded as %20, and some expect spaces encoded as +. For example, IMDB will only accept +. Any %20 is re-encoded into %2520 and redirected, which makes it impossible to use spaces in searches that will be sent to the IMDB. (Right now, I'm just trying to remember to type spaces as + by hand, since that works. Heh.) Thanks!

**Comment by rs** [comment]
Do you know how other browsers handle this issue?

**Comment by indirect** [comment]
No, sorry. I've only set up keywords in Safari before, and Keywurl (for example) had a per-keyword option to choose between + and %20.
